src/docs/getting-started/installation.md

```diff
@@ -4,66 +4,40 @@ description: "How to install probe-rs."
 order: 20
 ---
 
+# Installing a precompiled binary (`cargo binstall`)
 
-# Prerequisites
+## Prerequisites
 
-To build probe-rs, you will need a working Rust toolchain. The easiest way is to follow the [rustup installation instructions](https://rustup.rs).
+First, set up `cargo-binstall`. You can find setup instructions [`here`](https://crates.io/crates/cargo-binstall).
 
-Additionally, probe-rs depends on [libusb](https://libusb.info/) and optionally on [libftdi](https://www.intra2net.com/en/developer/libftdi/) libraries, which need to be installed to build probe-rs. Instructions for your OS are provided below.
+## Installation
 
-## Debian-based Linux
+`cargo binstall probe-rs`
```

Comment:
Does it not need the `--features cli` flag?

Reply:
I don't think it does
![image](https://github.com/probe-rs/webpage/assets/977627/3bcdaac6-b9c6-4510-b337-17a8a9756401)